server/src/main/java/io/openbytes/odi/domain/ODIResource.java

```diff
@@ -0,0 +1,28 @@
+package io.openbytes.odi.domain;
+
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * this is the abstraction of remote object info.
+ */
+
+@Getter
+@ToString
+public class ODIResource {
```

Comment:
is the ODI prefix necessary?
since you are already in odi.domain.

and I think odi resources are more then files, so suggest to rename ODIResource to File or Data.

Reply:
File looks good to me.

Reply:
Updated to DatasetFile